format/Schema.fbs

```diff
@@ -44,9 +44,14 @@ table Null {
 table Struct_ {
 }
 
+/// List with 32-bit offsets
 table List {
 }
 
+// List with 64-bit offsets
+table LargeList {
```

Comment:
Please document that this is type optional for implementations (unless we can get consensus on the mailing list if we want it to be globally supported)

Reply:
good point, I fixed it!